android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeViewModel.kt

```diff
@@ -79,6 +77,7 @@ class PracticeViewModel(
                         "entered = $enteredDots, expected = $expectedDots"
             )
         }
+    private var _hintUsed:Boolean = false

```

Comment:
Use autoformatting always on files you changed before commit

Reply:
done